core/src/utils/config.ts

```diff
@@ -204,6 +204,14 @@ export interface IonicConfig {
    */
   platform?: PlatformConfig;
 
+  /**
+   * @experimental
+   * If `true`, the [CloseWatcher API](https://github.com/WICG/close-watcher) will be used to handle
+   * all Escape key and hardware back button presses to dismiss menus and overlays and to navigate.
+   * Note that the `hardwareBackButton` config option must also be `true`.
```

Comment:
Optional: Could we log a warning if the CloseWatcher config is used but `hardwareBackButton` is still false?

Reply:
Good idea! Added in https://github.com/ionic-team/ionic-framework/pull/28705/commits/f34e136df164854d6e9189ad9f2250e972f45539